grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java

```diff
@@ -65,32 +79,41 @@
 @Api(value = "/tasks", description = "Endpoints used to query and control queued background tasks.", produces = "application/json")
 public class TasksController {
 
+    private static final Logger LOG = LoggerFactory.getLogger(GraknVertexProgram.class);
     private static final TaskState.Priority DEFAULT_TASK_PRIORITY = TaskState.Priority.LOW;
+
+    private static final int MAX_THREADS = 10;
+    private static final Duration MAX_EXECUTION_TIME = Duration.ofSeconds(10);
+
     private final TaskManager manager;
+    private final ExecutorService executor;
 
     public TasksController(Service spark, TaskManager manager) {
         if (manager==null) {
             throw GraknServerException.internalError("Task manager has not been instantiated.");
         }
         this.manager = manager;
 
-        spark.get(TASKS,       this::getTasks);
-        spark.get(GET,         this::getTask);
-        spark.put(STOP,        this::stopTask);
-        spark.post(TASKS,      this::createTask);
+        spark.get(TASKS, this::getTasks);
+        spark.get(GET, this::getTask);
+        spark.put(STOP, this::stopTask);
+        spark.post(TASKS, this::createTask);
+        spark.post(TASKS_BULK, this::createTaskBulk);
```

Comment:
At this stage I think  we can still afford rapid changes. We don't have any external reliance on the task controller so there should be no harm in killing it.